daffodil-lib/src/main/scala/org/apache/daffodil/externalvars/Binding.scala

```diff
@@ -45,6 +45,9 @@ class Binding(val varQName: RefQName, val varValue: String, scope: scala.xml.Nam
       GlobalQName(varQName.prefix, varQName.local, varQName.namespace)
 }
 
+case class BindingException(message: String)
+  extends Exception("Exception when processing external variable binidng file: %s".format(message))
```

Comment:
Typo "binidng"

Reply:
Also, a Binding doesn't necessarily need to come from a "file". Like it could from an API or the CLI. I imagine an error in one of those would still cause this Exception? Maybe just remove the work "file" if that's the case.